src/topics/group/group.api.test.ts

```diff
@@ -1,40 +1,55 @@
+import "reflect-metadata";
 import request from "supertest";
-import app from "../../api/app";
-import resetTestInfrastructure from "../../infrastructure/test-infrastructure";
-import testGroups from "./test-groups";
+import { DependencyContainer } from "tsyringe";
+import { getFastify } from "../../api/app";
+import { getTestGroups } from "./test-groups";
+import { FastifyInstance } from "fastify";
+import { getMemoryContainer } from "../../infrastructure";
+import { Group } from "./group";
 
 describe("test groups api", () => {
+  let fastify: FastifyInstance;
+  let container: DependencyContainer;
+  let testGroups: { [name: string]: Group };
+
+  beforeAll(() => {
+    container = getMemoryContainer();
+  });
+
   beforeEach(async () => {
-    await resetTestInfrastructure();
+    container.clearInstances();
+    fastify = getFastify(false, {}, container);
+    await fastify.ready();
+    testGroups = getTestGroups(container);
```

Comment:
Why "get" whereas we have a `groups[groupName] = Group.create(container, testGroups[groupName]);` inside it?
Maybe "createTestGroups"

Reply:
replaced !